db/models.py

```diff
@@ -50,3 +54,67 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ("-created_at",)
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = ForeignKey(MovieSession, on_delete=models.CASCADE)
```

Comment:
related names are missing  + try to be consistent with imports - in this module you use `models.ForeignKey` but only in this line you use `ForeignKey` which also added an extra import. It is not a mistake but it is better to be consistent